eZ/Bundle/EzPublishCoreBundle/EventListener/ViewControllerListener.php

```diff
@@ -66,6 +67,7 @@ public function getController(ControllerEvent $event)
         $parameterEvent = new FilterViewBuilderParametersEvent(clone $request);
         $this->eventDispatcher->dispatch($parameterEvent, ViewEvents::FILTER_BUILDER_PARAMETERS);
         $view = $viewBuilder->buildView($parameterEvent->getParameters()->all());
+        $this->eventDispatcher->dispatch(new BuildViewEvent($view));
```

Comment:
```suggestion
        $this->eventDispatcher->dispatch(new PostBuildViewEvent($view));
```

IMO would suit it better.